### PR TITLE
Update native flow `reqcnf` format

### PR DESCRIPTION
**Pull request by sameerag**
The `reqcnf` format is expected to be a short-form with base64 encoding for native flows for `POP` to work.

### CODE REVIEW

File: change/@azure-msal-browser-88c06603-4e47-4cc5-8d76-6aa3750c86de.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix pop format passed to STS in native flows (#6065)",
```
Comment (tnorling) [inline change/@azure-msal-browser-88c06603-4e47-4cc5-8d76-6aa3750c86de.json]: nit: Some of the linking doesn't work properly when wrapped in parens

```suggestion
  "comment": "Fix pop format passed to STS in native flows #6065",
```

File: change/@azure-msal-common-90a4c1f3-7a8d-45bd-8d6d-cb453c90a8b6.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix pop format passed to STS in native flows (#6065)",
```
Comment (tnorling) [inline change/@azure-msal-common-90a4c1f3-7a8d-45bd-8d6d-cb453c90a8b6.json]:
```suggestion
  "comment": "Fix pop format passed to STS in native flows #6065",
```

**Review by tnorling — APPROVED** [review]

**Review by konstantin-msft — APPROVED** [review]

**Comment by ghost** [comment]
Reminder: The next release is scheduled for next week and this PR appears to be stale :( 

If changes have been requested please address feedback. 
If this PR is still a work in progress please mark as draft.

**Comment by sameerag** [comment]
Closing in favor of #6357

**Pull request closed without merging**